### DISCUSSION TITLE
attributeの扱い

### DISCUSSION THREAD

**Discussion by TANAKA-Makoto** [onclouds-lab/raas-front-OAS#57] [Ideas]
#54

**Comment by maShimizu** [comment]
ロボットが対応していないAttributeは,
AttributeのないWaypointとして処理される.

**Comment by tanihiroki** [comment]
https://github.com/onclouds-lab/raas-front-OAS/issues/54#issuecomment-988732985

具体的には、次期バージョンでは、どう管理するか？をdiscussionするスレッド
